File: src/main/java/com/vjiki/music/dto/FollowerResponse.java
package com.vjiki.music.dto;

import java.time.OffsetDateTime;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class FollowerResponse {
    private UUID followerId;
    private String followerEmail;
    private String followerNickname;
    private String followerAvatarUrl;
    private OffsetDateTime followedAt;
}

File: src/main/kotlin/com/vjiki/music/dto/FollowerResponse.kt
package com.vjiki.music.dto

import java.time.OffsetDateTime
import java.util.UUID

data class FollowerResponse(
    val followerId: UUID,
    val followerEmail: String,
    val followerNickname: String,
    val followerAvatarUrl: String?,
    val followedAt: OffsetDateTime?
)

